client/extensions/woocommerce/app/settings/payments/payment-method-item.js

```diff
@@ -81,6 +82,16 @@ class PaymentMethodItem extends Component {
 			enabled
 		);
 
+		if ( enabled ) {
+			analytics.tracks.recordEvent( 'calypso_woocommerce_payment_method_enabled', {
+				paymentMethod: method.id,
```

Comment:
In this context, is id a number or a string like 'paypal' ? (Hoping it is the latter)